haystack/modeling/model/language_model.py

```diff
@@ -1646,11 +1646,11 @@ def forward(
         """
         Perform the forward pass of the DebertaV2 model.
 
-        :param input_ids: The ids of each token in the input sequence. Is a tensor of shape [batch_size, max_seq_len]
-        :param padding_mask: A mask that assigns a 1 to valid input tokens and 0 to padding tokens
-           of shape [batch_size, max_seq_len]
-        :param output_hidden_states: Whether to output hidden states in addition to the embeddings
-        :param output_attentions: Whether to output attentions in addition to the embeddings
+        :param input_ids: The IDs of each token in the input sequence. Is a tensor of shape [batch_size, max_seq_len].
+        :param padding_mask: A mask that assigns 1 to valid input tokens and 0 to padding tokens
+           of shape [batch_size, max_seq_len].
+        :param output_hidden_states: When set to `True`, outputs hidden states in addition to the embeddings.
+        :param output_attentions: When set to `True`, outputs attentions in addition to the embeddings.
```

Comment:
Same as above.

Reply:
Reviewed the doc strings for all models.